proto/astria/sequencer/v1alpha1/abci.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package astria.sequencer.v1alpha1;
+
+enum AbciErrorCode {
+  ABCI_ERROR_CODE_UNSPECIFIED = 0;
+  ABCI_ERROR_CODE_UNKNOWN_PATH = 1;
+  ABCI_ERROR_CODE_INVALID_PARAMETER = 2;
+  ABCI_ERROR_CODE_INTERNAL_ERROR = 3;
+  ABCI_ERROR_CODE_INVALID_NONCE = 4;
+  ABCI_ERROR_CODE_TRANSACTION_TOO_LARGE = 5;
```

Comment:
@noot I renamed this from `INVALID_SIZE` to make it slightly more specific/descriptive.